src/main/java/com/salesforce/dev/framework/DataDrivenManager.java

```diff
@@ -49,4 +51,92 @@ public DataDrivenManager(){
         return accountsArray.iterator();
     }
 
+    public Iterator<ViewSalesForce[]> getDataView(String fileJson){
+
+        Collection<ViewSalesForce[]> viewSalesForcesArray =new ArrayList<ViewSalesForce[]>();
+        try {
+            parser = new JSONParser();
+            String pathFileJson = "src\\test\\resources\\" + fileJson;
```

Comment:
Try to use absolute paths in order to avoid problems when executing on Windows, Linux and MacOS